src/components/DetailsAndMap/Components/DataTabs.tsx

```diff
@@ -0,0 +1,381 @@
+                <BorderDiv />
+                <DIV>
+                    <TabDataInfo>
+                        <DataContainer>
+                            <HomeIconDiv><HomeIcon style={{ color: "#318CE7" }} /> </HomeIconDiv>
+                            <DetailDiv>
+                                <DetailHeading>Entire Apartment</DetailHeading>
+                                <DetailDesc>4 Guests 1 bedroom 4 Guests 1 bedroom</DetailDesc>
+                            </DetailDiv>
+                        </DataContainer>
+                        <DataContainer>
+                            <HomeIconDiv><HomeIcon style={{ color: "#318CE7" }} /> </HomeIconDiv>
+                            <DetailDiv>
+                                <DetailHeading>Entire Apartment</DetailHeading>
+                                <DetailDesc>4 Guests 1 bedroom 4 Guests 1 bedroom</DetailDesc>
+                            </DetailDiv>
+                        </DataContainer>
+                        <DataContainer>
+                            <HomeIconDiv><HomeIcon style={{ color: "#318CE7" }} /> </HomeIconDiv>
+                            <DetailDiv>
+                                <DetailHeading>Entire Apartment</DetailHeading>
+                                <DetailDesc>4 Guests 1 bedroom 4 Guests 1 bedroom</DetailDesc>
+                            </DetailDiv>
+                        </DataContainer>
+                        <Amenties>
+                            <AmentiesTitle>Amenities</AmentiesTitle>
+                            <AcDiv>
+                                <Ac>
+                                    <AcUnitIcon style={{ color: "black" }} />
+                                    <AcText>Air condition</AcText>
+                                </Ac>
+                                <Ac>
+                                    <AcUnitIcon style={{ color: "black" }} />
+                                    <AcText>Air condition</AcText>
+                                </Ac>
+                            </AcDiv>
+                            <AcDiv>
+                                <Ac>
+                                    <AcUnitIcon style={{ color: "black" }} />
+                                    <AcText>Air condition</AcText>
+                                </Ac>
+                                <Ac>
+                                    <AcUnitIcon style={{ color: "black" }} />
+                                    <AcText>Air condition</AcText>
+                                </Ac>
+                            </AcDiv>
+                        </Amenties>
+                    </TabDataInfo>
+                    <TabDataReview>
+                        <StarDiv>
+                            <StarTitleDiv>
+                                <StarTitle>4 Reviews</StarTitle>
+                            </StarTitleDiv>
+                            <StarIconDiv>
+                                <StarIcon style={{ color: "gold", marginLeft: "5px" }} />
```

Comment:
please create reuseable component instead of repeating the code

Reply:
changed